tests/test_tablib.py

```diff
@@ -990,6 +991,15 @@ def test_tsv_export(self):
         self.assertEqual(tsv, self.founders.tsv)
 
 
+class ODSTests(BaseTestCase):
+    def test_ods_export_datatypes(self):
+        date_time = datetime.datetime(2019, 10, 4, 12, 30, 8)
+        data.append(('string', '004', 42, 21.55, Decimal('34.5'), date_time))
+        data.headers = ('string', 'start0', 'integer', 'float', 'decimal', 'date/time')
+        # ODS is currently write-only, just test that output doesn't crash.
+        data.ods
```

Comment:
Can we assert something on this?

Even if it's something like `assert data.ods is not None` and/or `assert len(data.ods)`?

```suggestion
        assert data.ods is not None
        assert len(data.ods)
```